Airport Ticket Booking System/BookingSystemUtils/InputValidation.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace Airport_Ticket_Booking_System;
+
+public class InputValidation : IInputValidation
```

Comment:
`InputValidator`. And would be nice to separate this validator to multiple classes based on the related validations. i.e. FlightValidator, EmailValidator, ...